index.html

```diff
@@ -111,6 +109,37 @@ <h2>Professional Software Development in 2018</h2>
           hodor - hodor hodor hodor. Hodor. Hodor hodor... Hodor hodor hodor hodor hodor? </p>
         <span class='expandButton'></span>
       </div>
+      <div class="article">
+        <h2>Why jquery will bring world peace</h2>
+        <p class="date">Nov 5th, 2017</p>
+
+        <p>Because jquery is the biggest and greatest creation of mankind. Because jquery is the biggest and greatest creation of mankind. 
+          Because jquery is the biggest and greatest creation of mankind. Because jquery is the biggest and greatest creation of mankind. 
+          Because jquery is the biggest and greatest creation of mankind. Because jquery is the biggest and greatest creation of mankind. 
```

Comment:
you have wrong text inside P tags - you might want to change it to - `JQUERY WILL DIE A SILENT DEATH`